.rubocop.yml

```diff
@@ -76,6 +76,7 @@ Rails/ThreeStateBooleanColumn:
     - db/migrate/2022*
     - db/migrate/20230*
     - db/migrate/202310*
+    - db/migrate/202312*
```

Comment:
I put these in as path exemptions because I thought we might want to avoid three-state booleans in future but that it wouldn't make sense to try to modify the existing migrations to fix this.

If we're happy to continue to have new nullable boolean columns, then I think we should just disable this rule entirely, rather than add new exemptions. Or, otherwise, perhaps only disable it with a comment in the file in question.